doc/api/process.markdown

```diff
@@ -240,12 +241,11 @@ Note:
 - `SIGKILL` cannot have a listener installed, it will unconditionally terminate
   Node.js on all platforms.
 - `SIGSTOP` cannot have a listener installed.
-- Sending `SIGINT`, `SIGTERM`, and `SIGKILL` cause the unconditional exit of the
-  target process.
 
 Note that Windows does not support sending Signals, but Node.js offers some
 emulation with `process.kill()`, and `child_process.kill()`. Sending signal `0`
-can be used to test for the existence of a process
+can be used to test for the existence of a process. Sending `SIGINT`,
+`SIGTERM`, and `SIGKILL` cause the unconditional exit of the target process.
```

Comment:
My bad, thanks. It does sound to me then that inevitably, Node should strive to unify the behavior. If we can't get Windows to behave, perhaps we need to downgrade the POSIX behavior? (that would suck, but if that's the only way ...)